lib/jnpr/junos/utils/sw.py

```diff
@@ -945,8 +946,9 @@ def _progress(report):
     # -------------------------------------------------------------------------
     # reboot - system reboot
     # -------------------------------------------------------------------------
-
-    def reboot(self, in_min=0, at=None, all_re=True, on_node=None):
+    @timeoutDecorator
+    def reboot(self, in_min=0, at=None, all_re=True, on_node=None,
+               vmhost=False, **kwargs):
```

Comment:
Why is `**kwargs` added, where is it used in the function ??

Reply:
Fixed via https://github.com/Juniper/py-junos-eznc/pull/952/commits/fc6cf16a7c66a5ac0d950b920e59bf698a51295e